main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"image/color"
+	"log"
+	"math"
+
+	"github.com/hajimehoshi/ebiten/v2"
+)
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+)
+
+type Line struct {
+	X1, Y1    int
+	Magnitude int
+	Degrees   float64
+	color.Color
+}
+
+func DrawLine(screen *ebiten.Image, x1, y1, x2, y2 float64, c color.Color) {
+	if x2 < x1 {
+		x1, x2 = x2, x1
+		y1, y2 = y2, y1
+	}
+	Δx, Δy := x2-x1, y2-y1
```

Comment:
Generally it is against the style guide to use non-alphanumeric characters in variable names.